src/test/request.test.js

```diff
@@ -313,7 +313,7 @@ describe('POST /api/v1/request', () => {
         from: ['Lagos', 'warri'],
         to: ['warri', 'kogi'],
         departureDate: ['2018-03-29T13:34:00.000', '2019-03-29T13:20:00.000'],
-        accommodation: ['hotel presidential', 'my house']
+        accommodation: ['hotel presidential']
```

Comment:
Does this solve the failing test issue? @codecell

Reply:
Yeah this partly solves it plus some things i removed from the initial file were re-added, i also removed them from the middleware/validateTripRequest.js